dev/drivers/scripts/prep/global_ens/jevs_global_ens_atmos_prep.sh

```diff
@@ -4,7 +4,7 @@
 #PBS -q dev
 #PBS -A VERF-DEV
 #PBS -l walltime=06:00:00
-#PBS -l place=vscatter,select=1:ncpus=83:mem=350GB
+#PBS -l place=vscatter:exclhost,select=2:ncpus=42:mem=500GB
```

Comment:
Following the PR conversation, should this line be reverted? NCO's installation has:
`#PBS -l place=vscatter,select=1:ncpus=83:mem=350GB`

Reply:
Yes, revert that.  The dev driver in the tag we sent to EVS has the original line in there, even though the ecf file has the correct resources.  But in order to test the resources in the dev driver, we'll need to have this line reverted back.